src/chainer.test.js

```diff
@@ -2,0 +1,85 @@
+const { chainer } = require("./chainer");
+
+describe("chainer", () => {
+  let plus, minus, multip, divid;
+
+  beforeEach(() => {
+    plus = (x) => x + 2;
+    minus = (x) => x - 1;
+    multip = (x) => x * 4;
+    divid = (x) => x / 2;
+  });
+
+  it("should chain functions correctly", () => {
+    const result = chainer([plus, minus, multip, divid])(2);
+    expect(result).toBe(6);
+  });
+  it("should works on fractials", () => {
+    const result = chainer([plus, minus, multip, divid])(8.8);
+    expect(result).toBe(19.6);
+  });
+  it("should works on negative numbers", () => {
+    const result = chainer([plus, minus, multip, divid])(-2);
+    expect(result).toBe(-2);
+  });
+  it("should chain functions with modified values", () => {
+    const result = chainer([
+      (x) => plus(x, 5),
+      (x) => minus(x, 3),
+      (x) => multip(x, 2),
+      (x) => divid(x, 4),
+    ])(7);
+
+    expect(result).toBe(16);
+  });
+  it("should return initial number no function are provided", () => {
```

Comment:
Typo in the test description: 'no function are provided' should be 'no functions are provided'.